packages/wrangler/src/triggers/deploy.ts

```diff
@@ -348,7 +351,7 @@ async function subdomainDeploy(
 		);
 	}
 
-	if (config.preview_urls == undefined && !previewsInSync) {
+	if (!firstDeploy && config.preview_urls == undefined && !previewsInSync) {
```

Comment:
Can we add an e2e test to confirm whether the warning is logged only after first deploy?